src/pages/api/lightning/withdraw.ts

```diff
@@ -0,0 +1,18 @@
+import axios, { AxiosResponse } from "axios";
+import { NextApiRequest, NextApiResponse } from "next";
+
+export default async function LightningWithdraw(
+  req: NextApiRequest,
+  res: NextApiResponse
+) {
+  const amount = req.body.amount ?? 0;
```

Comment:
We shouldn't create a zero-amount invoice. We're not doing any zero-amount validation in the client so we can throw a 400 error here if the amount is `0`
```suggestion
  const amount = req.body.amount ?? 0;
```